src/core/app/features.ts

```diff
@@ -22,4 +25,5 @@ export function setFeatures(value: BackendFeature[] | null) {
     newFeatures[ft.name] = true;
   });
   featureFlagsAtom.set.dispatch(newFeatures);
+  featuresWereAppliedAtom.setTrue.dispatch();
```

Comment:
1. When you need dispatch few actions - batch it.
```
     store.dispatch([
          featureFlagsAtom.set(newFeatures),
          featuresWereAppliedAtom.setTrue(),
        ]);
   ```
   
2. the function now does not simply set features, function name must be changed according to this changes

Reply:
It sets features and stores that it did. I'm not sure i need to change function name.
I've changed atom name that stores it